packages/browserslist-config-edu/index.js

```diff
@@ -0,0 +1,14 @@
+// See our browser support policy at https://go.czi.team/edu-browser-support (only accessible
+// by CZI employees).
+module.exports = [
+  'safari >= 14.1',
+  'ios_saf >= 14.5',
+
+  // Chrome must be at least `minimum_chrome_version` in Learning Platform's
+  // "assessment-lockdown/src/manifest.json"
+  'chrome >= 88',
+  'and_chr >= 88',
+
+  'edge >= 84',
+  'firefox >= 69',
```

Comment:
Updating our browser support will now require changing it here, publishing a new version, bringing the published version into whatever apps we want to update.

Maybe kind of annoying, but I figure it's infrequent enough that it's not that much of a concern?